### DISCUSSION TITLE
Depoy BoltCMS to Laravel Valet

### DISCUSSION THREAD

**Discussion by bingalls** [bolt/core#2775] [Q&A]
I'd like to set up Bolt CMS in a htdocs directory, to best simulate a shared server.
I use Laravel Valet, which makes it easy to create virtual hosts on a development box running nginx.
I also plan on using Bolt to serve GraphQL.
The standard install, with `bin/console server:start` worked well. However, I could not serve just the /public/ directory.
Next, I tried `composer create-project bolt/project bolt --prefer-dist` but I could get nothing to serve.
In all cases, I am getting an error
`syntax error, unexpected token "match"` ParseError in `vendor/willdurand/negotiation/src/Negotiation/Negotiator.php (line 41)`
DebugClassLoader->loadClass('Negotiation\\Negotiator')
in /Users/bruceingalls/Sites/bolt/vendor/symfony/config/Resource/ClassExistenceResource.php class_exists (line 81)

Note that I did do a `composer dump-autoload`

**Comment by bingalls** [comment]
OK, I found that my problem is in downgrading PHP 8 to BoltCMS's required PHP 7.
Valet does offer help, when Php-fpm is a different version from Php-cli, including a .valetphprc config for doc root,

**Comment by bobdenotter** [comment]
You can also use the bolt 5 beta, which _does_ work on PHP 8: 

https://github.com/bolt/project/releases/tag/2.1.0-beta.13

composer create-project bolt/project myprojectname "^2.0" --stability=beta
